php/ng/fpm/config.sls

```diff
@@ -11,6 +11,39 @@
   {% endif %}
 {% endfor %}
 
+
+{% if salt['pillar.get']('php:ng:version') is iterable %}
+  {% for version in salt['pillar.get']('php:ng:version') %}
+    {% set conf_settings = odict(php.lookup.fpm.defaults) %}
+    {% set first_version = salt['pillar.get']('php:ng:version')[0]|string %}
+    {% set ini = php.lookup.fpm.ini|replace(first_version, version) %}
+    {% set conf = php.lookup.fpm.conf|replace(first_version, version) %}
+    {% set pools = php.lookup.fpm.pools|replace(first_version, version) %}
+
+    {% for key, value in conf_settings.items() %}
+      {% if value is string %}
+        {% do conf_settings.update({key: value.replace(first_version, version)}) %}
+      {% endif %}
+    {% endfor %}
+    {% do conf_settings.global.update({'pid': '/var/run/php' + version + '-fpm.pid' }) %}
+    {% do conf_settings.global.update({'error_log': '/var/log/php' + version + '-fpm.log' }) %}
```

Comment:
not very satisfied with this part, I would want the loop above to be more generic but for some reason I couldn't get it to change the subdictionnary